dashpay/dash
# backport: bitcoin#15202, #15768, #16432, #17597 (Qt dash core wallet improvements)

## What was done?
qt-related backports from bitcoin with new features:
 - Close all wallets
 - Indiscreet (privacy) Overview page
 - Send QR code improvements
 - Close Window by hotkey (Ctrl + W)

Backports:
 - bitcoin/bitcoin#17597
 - bitcoin/bitcoin#15768
 - bitcoin/bitcoin#16432
 - bitcoin/bitcoin#15202
 

## How Has This Been Tested?
Run unit/functional tests

To test these functionality just open qt-dash and check it out!
 - enable/disable privacy mode
 - click "receive coins" and test functionality of dialog
 - click Close-All to be sure that wallets are indeed close.
 - open "console window" and press Ctrl + W (linux) - console would be closed
 
![image](https://github.com/dashpay/dash/assets/545784/28d65ad1-bee1-4df9-a822-e47adef3443e)
![image](https://github.com/dashpay/dash/assets/545784/ceee7aa6-c06a-423d-ae89-77d2899f51b0)
![image](https://github.com/dashpay/dash/assets/545784/03a2fb1e-9134-4d45-83af-73f6a76f1c7a)


## Breaking Changes
N/A

## Checklist:
